playbooks/roles/learner-profile-dashboard/templates/lpd_settings.py

```diff
@@ -0,0 +1,74 @@
+import getpass
+
+LTI_CLIENT_KEY = u'{{ LPD_LTI_CLIENT_KEY }}'
+LTI_CLIENT_SECRET = u'{{ LPD_LTI_CLIENT_SECRET }}'
+PASSWORD_GENERATOR_NONCE = u'{{ LPD_PASSWORD_GENERATOR_NONCE }}'
+
+DB_NAME = u'{{ LPD_DB_NAME }}'
+DB_USERNAME = u'{{ LPD_DB_USERNAME }}'
+DB_PASSWORD = u'{{ LPD_DB_PASSWORD }}'
+DB_HOST = u'{{ LPD_DB_HOST }}'
+
+DEBUG = False
+
+ALLOWED_HOSTS = ['{{ LPD_SERVER_DOMAIN }}']
+
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_SCHEME', 'https')
+
+
```

Comment:
@tomaszgy Extra blank line that we can remove.

Reply:
👍